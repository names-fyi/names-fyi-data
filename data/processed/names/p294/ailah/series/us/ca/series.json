[{"y":2018,"t":5,"m":0,"f":5,"mr":null,"fr":3234,"mp":0,"fp":0.00002642594393471735},{"y":2019,"t":6,"m":0,"f":6,"mr":null,"fr":2859,"mp":0,"fp":0.000032568339231821437},{"y":2020,"t":5,"m":0,"f":5,"mr":null,"fr":3192,"mp":0,"fp":0.00002877482548068346},{"y":2021,"t":7,"m":0,"f":7,"mr":null,"fr":2589,"mp":0,"fp":0.00004025001006250252}]