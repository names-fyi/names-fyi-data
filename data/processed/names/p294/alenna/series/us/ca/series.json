[{"y":2001,"t":5,"m":0,"f":5,"mr":null,"fr":3251,"mp":0,"fp":0.000022495467163366582},{"y":2010,"t":6,"m":0,"f":6,"mr":null,"fr":3175,"mp":0,"fp":0.00002844774859776306},{"y":2012,"t":6,"m":0,"f":6,"mr":null,"fr":3122,"mp":0,"fp":0.0000287079966124564},{"y":2014,"t":8,"m":0,"f":8,"mr":null,"fr":2526,"mp":0,"fp":0.000038183423621220435},{"y":2015,"t":5,"m":0,"f":5,"mr":null,"fr":3416,"mp":0,"fp":0.00002436730297817177},{"y":2016,"t":6,"m":0,"f":6,"mr":null,"fr":2995,"mp":0,"fp":0.000029479538743483795},{"y":2017,"t":5,"m":0,"f":5,"mr":null,"fr":3300,"mp":0,"fp":0.000025566685586014},{"y":2021,"t":8,"m":0,"f":8,"mr":null,"fr":2389,"mp":0,"fp":0.000046000011500002875}]