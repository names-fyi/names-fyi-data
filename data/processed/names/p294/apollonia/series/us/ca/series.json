[{"y":1986,"t":6,"m":0,"f":6,"mr":null,"fr":2297,"mp":0,"fp":0.000028891713856465966},{"y":2012,"t":5,"m":0,"f":5,"mr":null,"fr":3607,"mp":0,"fp":0.000023923330510380333},{"y":2013,"t":6,"m":0,"f":6,"mr":null,"fr":3052,"mp":0,"fp":0.00002926401014485685},{"y":2020,"t":7,"m":0,"f":7,"mr":null,"fr":2575,"mp":0,"fp":0.00004028475567295684},{"y":2022,"t":5,"m":0,"f":5,"mr":null,"fr":3271,"mp":0,"fp":0.00002899323877671727}]