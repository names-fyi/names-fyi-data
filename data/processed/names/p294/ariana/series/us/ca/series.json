[{"y":1965,"t":6,"m":0,"f":6,"mr":null,"fr":1477,"mp":0,"fp":0.000037114243828210536},{"y":1966,"t":10,"m":0,"f":10,"mr":null,"fr":1087,"mp":0,"fp":0.00006504741956886571},{"y":1968,"t":11,"m":0,"f":11,"mr":null,"fr":1096,"mp":0,"fp":0.00007151866637192307},{"y":1969,"t":6,"m":0,"f":6,"mr":null,"fr":1660,"mp":0,"fp":0.00003753307602324549},{"y":1970,"t":11,"m":0,"f":11,"mr":null,"fr":1178,"mp":0,"fp":0.00006706335658196361},{"y":1971,"t":10,"m":0,"f":10,"mr":null,"fr":1205,"mp":0,"fp":0.00006738453659654182},{"y":1972,"t":14,"m":0,"f":14,"mr":null,"fr":960,"mp":0,"fp":0.00010213610365355434},{"y":1973,"t":5,"m":0,"f":5,"mr":null,"fr":1812,"mp":0,"fp":0.00003823009932179804},{"y":1974,"t":11,"m":0,"f":11,"mr":null,"fr":1137,"mp":0,"fp":0.0000805182447022655},{"y":1975,"t":16,"m":0,"f":16,"mr":null,"fr":939,"mp":0,"fp":0.00011556602070076346},{"y":1976,"t":18,"m":0,"f":18,"mr":null,"fr":880,"mp":0,"fp":0.00012511468846442572},{"y":1977,"t":29,"m":0,"f":29,"mr":null,"fr":671,"mp":0,"fp":0.00019389174154899443},{"y":1978,"t":37,"m":0,"f":37,"mr":null,"fr":580,"mp":0,"fp":0.0002413174629055927},{"y":1979,"t":36,"m":0,"f":36,"mr":null,"fr":614,"mp":0,"fp":0.0002204153605015674},{"y":1980,"t":130,"m":0,"f":130,"mr":null,"fr":256,"mp":0,"fp":0.0007483866510848728},{"y":1981,"t":149,"m":0,"f":149,"mr":null,"fr":236,"mp":0,"fp":0.0008219283874206343},{"y":1982,"t":159,"m":0,"f":159,"mr":null,"fr":228,"mp":0,"fp":0.0008560629717768421},{"y":1983,"t":164,"m":0,"f":164,"mr":null,"fr":218,"mp":0,"fp":0.0008737300280765686},{"y":1984,"t":137,"m":0,"f":137,"mr":null,"fr":256,"mp":0,"fp":0.0007113668108439303},{"y":1985,"t":156,"m":0,"f":156,"mr":null,"fr":239,"mp":0,"fp":0.0007713950877956396},{"y":1986,"t":173,"m":5,"f":168,"mr":1798,"fr":226,"mp":0.000021901774919839505,"fp":0.0008089679879810471},{"y":1987,"t":184,"m":0,"f":184,"mr":null,"fr":210,"mp":0,"fp":0.0008554439242560195},{"y":1988,"t":228,"m":0,"f":228,"mr":null,"fr":179,"mp":0,"fp":0.0009982443159180564},{"y":1989,"t":313,"m":5,"f":308,"mr":2087,"fr":142,"mp":0.000018612754203690536,"fp":0.0012623882089662353},{"y":1990,"t":406,"m":0,"f":406,"mr":null,"fr":124,"mp":0,"fp":0.0015471912381721804},{"y":1991,"t":451,"m":0,"f":451,"mr":null,"fr":120,"mp":0,"fp":0.001724724751521085},{"y":1992,"t":418,"m":0,"f":418,"mr":null,"fr":127,"mp":0,"fp":0.0016279033539482498},{"y":1993,"t":400,"m":0,"f":400,"mr":null,"fr":134,"mp":0,"fp":0.001602724631874186},{"y":1994,"t":494,"m":0,"f":494,"mr":null,"fr":108,"mp":0,"fp":0.002037340239944241},{"y":1995,"t":516,"m":0,"f":516,"mr":null,"fr":104,"mp":0,"fp":0.0021997041483180365},{"y":1996,"t":543,"m":0,"f":543,"mr":null,"fr":93,"mp":0,"fp":0.00236332536853513},{"y":1997,"t":551,"m":0,"f":551,"mr":null,"fr":87,"mp":0,"fp":0.0024726926770600537},{"y":1998,"t":587,"m":0,"f":587,"mr":null,"fr":78,"mp":0,"fp":0.0026561566725189596},{"y":1999,"t":591,"m":0,"f":591,"mr":null,"fr":75,"mp":0,"fp":0.0026980018351890657},{"y":2000,"t":638,"m":0,"f":638,"mr":null,"fr":64,"mp":0,"fp":0.0028507596067917783},{"y":2001,"t":699,"m":0,"f":699,"mr":null,"fr":55,"mp":0,"fp":0.003144866309438648},{"y":2002,"t":744,"m":0,"f":744,"mr":null,"fr":57,"mp":0,"fp":0.0033407122362566063},{"y":2003,"t":713,"m":0,"f":713,"mr":null,"fr":59,"mp":0,"fp":0.003146055517069447},{"y":2004,"t":791,"m":0,"f":791,"mr":null,"fr":47,"mp":0,"fp":0.0034572282732215597},{"y":2005,"t":779,"m":0,"f":779,"mr":null,"fr":50,"mp":0,"fp":0.0033813112021668173},{"y":2006,"t":832,"m":0,"f":832,"mr":null,"fr":47,"mp":0,"fp":0.0035446489434219496},{"y":2007,"t":800,"m":0,"f":800,"mr":null,"fr":48,"mp":0,"fp":0.003386960203217612},{"y":2008,"t":729,"m":0,"f":729,"mr":null,"fr":51,"mp":0,"fp":0.0031797403866285156},{"y":2009,"t":634,"m":0,"f":634,"mr":null,"fr":59,"mp":0,"fp":0.002905178457689329},{"y":2010,"t":549,"m":0,"f":549,"mr":null,"fr":69,"mp":0,"fp":0.00260296899669532},{"y":2011,"t":629,"m":0,"f":629,"mr":null,"fr":51,"mp":0,"fp":0.003030668047893228},{"y":2012,"t":642,"m":0,"f":642,"mr":null,"fr":49,"mp":0,"fp":0.003071755637532835},{"y":2013,"t":748,"m":0,"f":748,"mr":null,"fr":38,"mp":0,"fp":0.0036482465980588207},{"y":2014,"t":905,"m":0,"f":905,"mr":null,"fr":25,"mp":0,"fp":0.004319499797150562},{"y":2015,"t":855,"m":0,"f":855,"mr":null,"fr":31,"mp":0,"fp":0.004166808809267373},{"y":2016,"t":786,"m":0,"f":786,"mr":null,"fr":35,"mp":0,"fp":0.003861819575396377},{"y":2017,"t":636,"m":0,"f":636,"mr":null,"fr":49,"mp":0,"fp":0.003252082406540981},{"y":2018,"t":666,"m":0,"f":666,"mr":null,"fr":40,"mp":0,"fp":0.0035199357321043507},{"y":2019,"t":667,"m":0,"f":667,"mr":null,"fr":40,"mp":0,"fp":0.0036205137112708167},{"y":2020,"t":565,"m":0,"f":565,"mr":null,"fr":47,"mp":0,"fp":0.003251555279317231},{"y":2021,"t":562,"m":0,"f":562,"mr":null,"fr":47,"mp":0,"fp":0.003231500807875202},{"y":2022,"t":518,"m":0,"f":518,"mr":null,"fr":49,"mp":0,"fp":0.0030036995372679093}]