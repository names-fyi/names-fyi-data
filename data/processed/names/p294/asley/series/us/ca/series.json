[{"y":1987,"t":5,"m":0,"f":5,"mr":null,"fr":2699,"mp":0,"fp":0.000023245758811304876},{"y":1989,"t":6,"m":0,"f":6,"mr":null,"fr":2677,"mp":0,"fp":0.00002459197809674484},{"y":1990,"t":8,"m":0,"f":8,"mr":null,"fr":2367,"mp":0,"fp":0.000030486526860535572},{"y":1991,"t":8,"m":0,"f":8,"mr":null,"fr":2365,"mp":0,"fp":0.00003059378716667113},{"y":1992,"t":6,"m":0,"f":6,"mr":null,"fr":2913,"mp":0,"fp":0.00002336703378873086},{"y":1993,"t":7,"m":0,"f":7,"mr":null,"fr":2566,"mp":0,"fp":0.000028047681057798256}]