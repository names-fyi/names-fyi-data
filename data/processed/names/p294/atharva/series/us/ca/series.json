[{"y":2001,"t":6,"m":6,"f":0,"mr":1933,"fr":null,"mp":0.00002452312735268753,"fp":0},{"y":2002,"t":5,"m":5,"f":0,"mr":2226,"fr":null,"mp":0.000020405248229844715,"fp":0},{"y":2004,"t":7,"m":7,"f":0,"mr":1861,"fr":null,"mp":0.000027767199796902767,"fp":0},{"y":2005,"t":8,"m":8,"f":0,"mr":1816,"fr":null,"mp":0.00003148131387263447,"fp":0},{"y":2007,"t":9,"m":9,"f":0,"mr":1723,"fr":null,"mp":0.000034426436443060585,"fp":0},{"y":2008,"t":11,"m":11,"f":0,"mr":1559,"fr":null,"mp":0.00004329072476544298,"fp":0},{"y":2009,"t":7,"m":7,"f":0,"mr":2049,"fr":null,"mp":0.000028916777514313806,"fp":0},{"y":2010,"t":8,"m":8,"f":0,"mr":1857,"fr":null,"mp":0.00003423104242081932,"fp":0},{"y":2011,"t":6,"m":6,"f":0,"mr":2292,"fr":null,"mp":0.000026056029148011274,"fp":0},{"y":2012,"t":7,"m":7,"f":0,"mr":2064,"fr":null,"mp":0.000030381812578938458,"fp":0},{"y":2013,"t":10,"m":10,"f":0,"mr":1587,"fr":null,"mp":0.00004406936518079457,"fp":0},{"y":2014,"t":7,"m":7,"f":0,"mr":2114,"fr":null,"mp":0.000030281007751937984,"fp":0},{"y":2015,"t":11,"m":11,"f":0,"mr":1550,"fr":null,"mp":0.0000486458757142099,"fp":0},{"y":2016,"t":6,"m":6,"f":0,"mr":2312,"fr":null,"mp":0.000026847559556836283,"fp":0},{"y":2017,"t":8,"m":8,"f":0,"mr":1904,"fr":null,"mp":0.000037124520281589484,"fp":0},{"y":2018,"t":10,"m":10,"f":0,"mr":1649,"fr":null,"mp":0.00004849002075372888,"fp":0},{"y":2019,"t":16,"m":16,"f":0,"mr":1223,"fr":null,"mp":0.00007890791446382072,"fp":0},{"y":2020,"t":18,"m":18,"f":0,"mr":1077,"fr":null,"mp":0.00009517816824327539,"fp":0},{"y":2021,"t":14,"m":14,"f":0,"mr":1301,"fr":null,"mp":0.00007420402927878984,"fp":0},{"y":2022,"t":9,"m":9,"f":0,"mr":1749,"fr":null,"mp":0.00004798234249796075,"fp":0}]