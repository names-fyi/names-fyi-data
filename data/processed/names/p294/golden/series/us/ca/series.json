[{"y":1979,"t":5,"m":5,"f":0,"mr":1491,"fr":null,"mp":0.000027817649742408563,"fp":0},{"y":2014,"t":13,"m":5,"f":8,"mr":2706,"fr":2591,"mp":0.000021629291251384276,"fp":0.000038183423621220435},{"y":2015,"t":5,"m":0,"f":5,"mr":null,"fr":3608,"mp":0,"fp":0.00002436730297817177},{"y":2017,"t":5,"m":5,"f":0,"mr":2667,"fr":null,"mp":0.00002320282517599343,"fp":0},{"y":2018,"t":7,"m":7,"f":0,"mr":2106,"fr":null,"mp":0.00003394301452761022,"fp":0},{"y":2019,"t":15,"m":8,"f":7,"mr":1931,"fr":2694,"mp":0.00003945395723191036,"fp":0.000037996395770458344},{"y":2020,"t":13,"m":7,"f":6,"mr":2037,"fr":2938,"mp":0.0000370137320946071,"fp":0.00003452979057682015},{"y":2021,"t":11,"m":5,"f":6,"mr":2632,"fr":2961,"mp":0.000026501439028139227,"fp":0.00003450000862500216},{"y":2022,"t":11,"m":5,"f":6,"mr":2656,"fr":2990,"mp":0.00002665685694331153,"fp":0.00003479188653206073}]